Jenkinsfile.upstream-operator-sync

```
@@ -0,0 +1,168 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+boolean goModModified = false
+operatorTagVersion = "upstream-sync"
+
+pipeline {
+    agent {
+        label 'kogito-operator-slave && !master'
+    }
+    // Needed for local build
+    tools {
+        jdk 'kie-jdk11'
+    }
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
+        timeout(time: 90, unit: 'MINUTES')
+    }
+    environment {
+        KOGITO_CI_EMAIL_TO = credentials("${JENKINS_EMAIL_CREDS_ID}")
+        OPENSHIFT_INTERNAL_REGISTRY = 'image-registry.openshift-image-registry.svc:5000'
+
+        // Use buildah container engine in this pipeline
+        CONTAINER_ENGINE="podman"
+        CONTAINER_TLS_OPTIONS="--tls-verify=false"
+    }
+    stages {
+        stage('Initialize') {
+            steps {
+                script{
+                    checkout(githubscm.resolveRepository("${env.REPO_NAME}", "${env.GIT_AUTHOR}", "${BUILD_BRANCH_NAME}", false))
+                }
+            }
+        }
+        stage('Check if Kogito operator has changed') {
+            steps {
+                sh 'make update-dependencies'
+                script {
+                    goModModified = githubscm.isThereAnyChanges()
+                }
```

I would include the `sh` command in `script` for better readability
```suggestion
                script {
                    sh 'make update-dependencies'
                    goModModified = githubscm.isThereAnyChanges()
                }
```